### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by mtmcgrew**
Simplifying query

### CODE REVIEW

**Comment by osqueryer** [comment]
Can one of the osquery admins reply with "ok to test" to kick off a build...

**Comment by mtmcgrew** [comment]
#impact

File: README.md [hunk 1]
```diff
@@ -51,10 +51,7 @@ AND
 
 Check for ARP anomalies from the host's perspective:
 ```sql
-SELECT address, mac, mac_count 
-FROM
-  (SELECT address, mac, count(mac) AS mac_count FROM arp_cache GROUP BY mac) 
-WHERE mac_count > 1;
+SELECT address, mac, count(mac) AS mac_count FROM arp_cache GROUP BY mac HAVING count(mac) > 1;
```
Comment (theopolis) [inline README.md]: I LOVE 'having', I had no idea it was supported in SQLite!

**Comment by theopolis** [comment]
Can you think of another example that doesn't need an aggregate group by, which demos the use of subqueries? This example was to highlight that we support subqueries, haha :P

**Comment by marpaia** [comment]
Moving tracking to #841

**Pull request closed without merging**